src/server/bg_services/log_replication_scanner.js

```diff
@@ -107,19 +109,30 @@ class LogReplicationScanner {
      * @param {*} src_bucket source bucket
      * @param {*} dst_bucket destination bucket
      * @param {nb.ReplicationLogCandidates} candidates 
+     * @param {boolean} sync_versions should we sync object versions
      */
-    async process_candidates(src_bucket, dst_bucket, candidates) {
+    async process_candidates(src_bucket, dst_bucket, candidates, sync_versions) {
+        return sync_versions ? this.process_candidates_sync_version(src_bucket, dst_bucket, candidates) :
+            this.process_candidates_not_sync_version(src_bucket, dst_bucket, candidates);
+    }
+
+    async process_candidates_sync_version(src_bucket, dst_bucket, candidates) {
```

Comment:
please give a comment here about what is written in the dbg.warn. that it's currently not supported but we are working on it (AC preparation :) )